design_system/shared/src/commonMain/kotlin/ui/composable/BpThreeDotLoadingIndicator.kt

```diff
@@ -0,0 +1,64 @@
+package com.beepbeep.designSystem.ui.composable
+
+import androidx.compose.animation.core.Animatable
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.beepbeep.designSystem.ui.theme.Theme
+import kotlinx.coroutines.delay
+
+@Composable
+fun BpThreeDotLoadingIndicator(
+    modifier: Modifier = Modifier,
+    dotSize: Dp = 12.dp,
+    dotPadding: Dp = 2.dp,
+    dotColor: Color = Theme.colors.onPrimary,
+    animationDelay: Int = 600,
+    initialAlpha: Float = 0.38f,
+) {
+
+    val circles = listOf(
+        remember { Animatable(initialValue = initialAlpha) },
+        remember { Animatable(initialValue = initialAlpha) },
+        remember { Animatable(initialValue = initialAlpha) }
```

Comment:
I guess there is a better way to handle this case using 1 Animatable 
Lets Assume we have one Animatable
Take value from initial alpha to one alpha
So may we can use this single animatble to handle all dots 
Like when first dot was in initial alpha the second dot was in initial alpha+someRatio.
May u could find better way than it. I trust U will ❤️.
Thanks for reading @devsadeq